src/main/java/com/minecraftabnormals/upgrade_aquatic/core/other/UADispenseBehaviorRegistry.java

```diff
@@ -128,8 +129,8 @@ protected ItemStack dispenseStack(IBlockSource source, ItemStack stack) {
                     }
                 }
             }
-            return new FishBucketDispenseBehavior().dispense(source, stack);
-        }
+            return stack;
+	}
```

Comment:
The formatting got a bit messed up here.

Reply:
Hmm yeah, that might be an intellij moment. I'll try to fix it.

Reply:
Wait no it looks like that was just a me moment. In any case, it's fixed (I think).

Reply:
Ok sorry about the wait i ended up getting swamped by school, i think it's fixed now - the changed files tab shows no difference in the indentation after the commit i just made